h2o-docs/src/product/data-science/algo-params/missing_values_handling.rst

```diff
@@ -7,7 +7,7 @@
 Description
 ~~~~~~~~~~~
 
-This option is used to specify the way that the algorithm will treat missing values. In H2O, the Deep Learning and GLM algorithms will either skip or mean-impute rows with NA values. This option defaults to MeanImputation. Note that in Deep Learning, unseen categorical variables are imputed by adding an extra “missing” level. In GLM, unseen categorical levels are replaced by the most frequent level present in training (mod). Optionally, either algorithm can skip all rows with any missing values.
+This option is used to specify the way that the algorithm will treat missing values. In H2O, the Deep Learning and GLM algorithms will either skip or mean-impute rows with NA values. The GLM algorithm can skip or mean-impute rows with NA values, or you can use plug_values, which allows you to specify a single-row frame that contains values that will be used to impute missing values of the training/validation frame. Both algorithms default to MeanImputation. Note that in Deep Learning, unseen categorical variables are imputed by adding an extra “missing” level. In GLM, unseen categorical levels are replaced by the most frequent level present in training (mod). Optionally, either algorithm can skip all rows with any missing values.
```

Comment:
This sounds a little redundant:
"... the Deep Learning and GLM algorithms will either skip or mean-impute rows with NA values. The GLM algorithm can skip or mean-impute rows with NA values,..."